talk/talk-api.yaml

```
@@ -2688,3 +2729,74 @@ components:
         - id
         - email
       description: User model
+
+    Widgets:
+      type: object
+      properties:
+        totalCalls:
+          $ref: '#/components/schemas/TotalCallsWidget'
+        missedCalls:
+          $ref: '#/components/schemas/MissedCallsWidget'
+        totalCallsTime:
+          $ref: '#/components/schemas/TotalCallsTimeWidget'
+        averageCallTime:
+          $ref: '#/components/schemas/AverageCallTimeWidget'  
+        liveCalls:
+          $ref: '#/components/schemas/LiveCallsWidget'  
+
+    TotalCallsWidget:
+      type: object
+      properties:
+        inbound: 
+          type: integer
+          format: int64
+        inboundDifference:
```

```suggestion
        inboundDelta:
```
I thing delta is the correct word when talking about difference over time

---

I think that delta is a more technical word, but the difference is more user friendly.